llvm/test/MC/X86/apx/ctest-intel.s

```diff
@@ -768,3 +768,54 @@
 # CHECK: cteste {dfv=of} r15, r9
 # CHECK: encoding: [0x62,0x54,0xc4,0x04,0x85,0xcf]
          cteste {dfv=of} r15, r9
+
+## "{evex} test*" are alias for "cteste* {dfv=of,sf,zf,cf}"
```

Comment:
Shouldn't be the alias of `ctestt`?

Reply:
yes, 1edff3ab2eab856d7720af248dc03b58ac5da4f8. pls review again.